layouts/shortcodes/observability_pipelines/prerequisites/kafka.md

```diff
@@ -0,0 +1,4 @@
+To use Observability Pipelines's Kafka source, you need the following information available:
```

Comment:
Just flagging that there's more than one instance of "Observability Pipelines's", beyond that I won't keep spamming you!

Reply:
Thanks, I must have copy/pasta'd it! I'll make a batch update.